README.md

```diff
@@ -1,81 +1,101 @@
 
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
 
-## Prerequisites
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
 
-### A Passkey Flex app
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
 
-1. Create a Passkey Flex app in the Passage Console at https://console.passage.id 
-2. Add an Android and/or iOS app in the Native Apps section
-3. When you add your Native App info, you can generate the associated domain file for that app if you haven’t already created it yourself, as shown below:
-![Passage Native Apps screenshot](https://docs.passage.id/_next/image?url=%2Fimages%2Fv1-doc-images%2Fios-download-config-file.png&w=3840&q=75)
 
 
-### Hosted associated domains files
+### In passage-flex-react-native
 
-In order for passkeys to work, you’ll need to associate your native app(s) with the public web domain you assigned to your Passkey Flex app.
+Use passage-flex-react-native to implement Passkey Flex in your Swift application to add native passkey authentication in your own authentication flows.
 
-Android requires an `assetlinks.json` file configured and hosted (learn more [here](https://developer.android.com/identity/sign-in/credential-manager#add-support-dal)).
 
-Apple requires an `apple-app-site-association` file configured and hosted (learn more [here](https://developer.apple.com/documentation/Xcode/supporting-associated-domains)).
+| Product | Compatible |
+| --- | --- |
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex** | ✅
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✖️ For Passkey Complete, check out [passage-react-native](https://github.com/passageidentity/passage-react-native)
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready** | ✖️ For Passkey Ready, check out [Authentikit for Android](https://github.com/passageidentity/passage-android/tree/main/authentikit) and [Authentikit for iOS](https://github.com/passageidentity/passage-authentikit-ios) |
 
-## Installation
+## Getting Started
 
-Install this package using npm:
+### Check Prerequisites
 
-```
-npm i --save @passageidentity/passage-flex-react-native
-```
+<p>
+ You'll need a free Passage account and a Passkey Flex app set up in <a href="https://console.passage.id/">Passage Console</a> to get started. <br />
+ <sub><a href="https://docs.passage.id/home#passage-console">Learn more about Passage Console →</a></sub>
+</p>
 
+Add an Android and/or iOS app in the Native Apps section.
```

Comment:
I think it makes sense to pull up these more detailed prereq steps into this section. We can delete them/edit what we have in the actual html template when we remove after we have this enshrined in docs.